packages/tools/client-debugger/client-debugger-view/src/Debugger.tsx

```diff
@@ -33,145 +45,149 @@ export interface FluidClientDebuggersProps {
-			<></>
-		);
-
-	return (
-		<Resizable
-			style={{
-				position: "absolute",
-				top: "0px",
-				right: "0px",
-				bottom: "0px",
-				zIndex: "2",
-				backgroundColor: "lightgray", // TODO: remove
-			}}
-			defaultSize={{ width: 400, height: "100%" }}
-			className={"debugger-panel"}
-		>
-			{selectionView}
-			{view}
-		</Resizable>
-	);
-}
-
-/**
- * Base props interface used by components below which can re-attempt to find the debugger instance
- * associated with some Container ID.
- */
-interface CanLookForDebugger {
-	/**
-	 * Retry looking for the debugger instance.
-	 */
-	onRetryDebugger(): void;
-}
-
-/**
- * {@link NoDebuggerInstance} input props.
- */
-type NoDebuggerInstanceProps = CanLookForDebugger;
+	}, [setContainers, messageRelay]);
+
+	let innerView: React.ReactElement;
+	switch (menuSelection?.type) {
+		case "telemetryMenuSelection":
+			innerView = <TelemetryView />;
+			break;
+		case "containerMenuSelection":
+			// eslint-disable-next-line no-case-declarations
+			const container = containers?.find((x) => x.id === menuSelection.containerId);
+			innerView =
+				container === undefined ? (
+					<div>Could not find a debugger for that container.</div>
+				) : (
+					<ContainerView containerId={menuSelection.containerId} />
+				);
+			break;
+		default:
+			innerView = <LandingView />;
+			break;
+	}
 
-function NoDebuggerInstance(props: NoDebuggerInstanceProps): React.ReactElement {
-	const { onRetryDebugger } = props;
+	// Styles definition
+	const stackStyles: IStackStyles = {
+		root: {
+			background: DefaultPalette.themeTertiary,
+			height: "100%",
+		},
+	};
+	const contentViewStyles: IStackItemStyles = {
+		root: {
+			alignItems: "center",
+			background: DefaultPalette.themeLight,
+			color: DefaultPalette.black,
+			display: "flex",
+			justifyContent: "center",
+		},
+	};
+
+	const menuStyles: IStackItemStyles = {
+		root: {
+			...contentViewStyles,
+			display: "flex",
+			flexDirection: "column",
+			borderRight: `1px solid ${DefaultPalette.themePrimary}`,
+			minWidth: 150,
+		},
+	};
+
+	function onContainerClicked(id: string): void {
+		setMenuSelection({ type: "containerMenuSelection", containerId: id });
+	}
 
-	const retryButtonTooltipId = useId("retry-button-tooltip");
+	function onTelemetryClicked(): void {
+		setMenuSelection({ type: "telemetryMenuSelection" });
+	}
 
-	// TODO: give more info and link to docs, etc. for using the tooling.
 	return (
-		<Stack horizontalAlign="center" tokens={{ childrenGap: 10 }}>
-			<StackItem>
-				<div>No Fluid Client debuggers found.</div>
-			</StackItem>
-			<StackItem>
-				<TooltipHost content="Look again" id={retryButtonTooltipId}>
-					<IconButton
-						onClick={onRetryDebugger}
-						menuIconProps={{ iconName: "Refresh" }}
-						aria-describedby={retryButtonTooltipId}
+		<Stack enableScopedSelectors horizontal styles={stackStyles}>
+			<Stack.Item grow={1} styles={menuStyles}>
+				{/* TODO: button to refresh list of containers */}
```

Comment:
TODO before merge, or sometime in the future?

Reply:
My intention was the short-term future :)

Reply:
Should we file a task to track this then?

Reply:
https://dev.azure.com/fluidframework/internal/_workitems/edit/3854, should be part of grooming tomorrow.